testscommon/components/components.go

```diff
@@ -576,15 +576,16 @@ func GetProcessArgs(
 		FlagsConfig: config.ContextFlagsConfig{
 			Version: "v1.0.0",
 		},
-		TxExecutionOrderHandler:               &commonMocks.TxExecutionOrderHandlerStub{},
+		TxExecutionOrderHandler: &commonMocks.TxExecutionOrderHandlerStub{},
+		// TODO: add mocks
```

Comment:
Would've been better here and other places to directly reference the MX-1... task from Jira for easier traceability